src/transformers/models/bamba/modeling_bamba.py

```diff
@@ -1258,6 +1312,11 @@ def forward(
                     use_cache=use_cache,
                     cache_position=cache_position,
                     position_embeddings=position_embeddings,
+                    cu_seq_lens_q=cu_seq_lens_q,
+                    cu_seq_lens_k=cu_seq_lens_k,
+                    max_length_q=max_length_q,
+                    max_length_k=max_length_k,
+                    seq_idx=seq_idx,
```

Comment:
why don't we just use kwargs everywhere?

Reply:
Do you mean pack all of these `forward` method padding free arguments into ` BambaModel.forward(..., *kwargs)` and just call 
```py
decoder_layer(
    hidden_states, 
    ...,
    **kwargs
)
```
here?

> Let's rather create a path (meaning support passing them as kwargs typed properly!)

I thought above you were asking me to enumerate all of these padding free args as individually type keyword arguments.